lib/prepare.js

```diff
@@ -2,50 +2,60 @@
 
 const execa = require('execa');
 const { readJson } = require('fs-extra');
+const path = require('path');
 const { isOvsxPublishEnabled, isTargetEnabled } = require('./utils');
 
-module.exports = async (version, packageVsix, logger, cwd) => {
-  if (packageVsix === false) {
+module.exports = async (pluginConfig, { logger, cwd, version }) => {
```

Comment:
If you pass in `cwd` from `index.js` already resolved, I'd appreciate if you revert all changes related to the function's declarations.

This changes a lot of files and makes it difficult to review, while not being directly related to the feature's addition.